assets/js/components/consent-mode/ConsentModeSetupCTABanner.js

```diff
@@ -94,11 +88,9 @@ export default function ConsentModeSetupCTABanner( { id, Notification } ) {
 
 		const promises = [ saveConsentModeSettings() ];
 
-		if ( usingProxy ) {
-			promises.push(
-				triggerSurvey( 'enable_como', { ttl: DAY_IN_SECONDS } )
-			);
-		}
+		promises.push(
+			triggerSurvey( 'enable_como', { ttl: DAY_IN_SECONDS } )
+		);
```

Comment:
Since this is no longer conditional, it can be merged into the array above which can be inlined into the `Promise.all` call since the array never changes now.